plugins/lsp/language-servers/nil_ls-settings.nix

```
@@ -1,13 +1,19 @@
 { lib, helpers }:
-# Options:
-#  - https://github.com/oxalica/nil/blob/main/docs/configuration.md?plain=1#
+# All available settings are documented here:
+# https://github.com/oxalica/nil/blob/main/docs/configuration.md
 with lib;
 {
   formatting = {
-    command = helpers.defaultNullOpts.mkListOf types.str null ''
-      External formatter command (with arguments).
-      It should accepts file content in stdin and print the formatted code into stdout.
-    '';
+    command = helpers.defaultNullOpts.mkListOf' {
+      type = types.str;
+      default = null;
+      description = ''
+        External formatting command, complete with required arguments).
```

```suggestion
        External formatting command, complete with required arguments.
```

This might've been my fault: this parentheses should go too

---

Definitely